core/src/main/java/bisq/core/account/score/ScoreInfo.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.core.account.score;
+
+import org.bitcoinj.core.Coin;
+
+import java.util.Optional;
+
+import lombok.Value;
+
+@Value
+public class ScoreInfo {
+    private AccountScoreCategory accountScoreCategory;
+    private long accountAge;
+    private Optional<Long> signedTradeAge;
+    private Coin minDepositAsCoin; //TODO remove?
```

Comment:
I think this will be gone as we are not able to force this from the maker's side - right?

Reply:
Probably